### PR TITLE
Anonymous functions

### LINKED ISSUE #1090: Anonymous functions
In dev mode React returns: "Anonymous arrow functions cause Fast Refresh to not preserve local component state", It's a not so potential warning but I want to send PR to Make all anonymous functions to Named arrow functions. Can I take its PR?

### PR DESCRIPTION
**Pull request by Aslemammad**
Fixes #1090

### CODE REVIEW

**Comment by Aslemammad** [comment]
@kavu

File: components/Billing.js [hunk 1]
```diff
@@ -144,125 +144,127 @@ function Billing(props) {
...
+        `}
+      </style>
     </div>
   )
 }
 
 const BillingWithStripe = injectStripe(Billing)
 
-export default () => {
+const ExportedBilling = () => {
```
Comment (mfix22) [inline components/Billing.js:267]: Please use: 
```suggestion
export default function ExportedBilling() {
```

File: components/style/Font.js [hunk 2]
```diff
@@ -5,7 +5,7 @@
  */
 import React from 'react'
 
-export default () => (
+const Font = () => (
```
Comment (mfix22) [inline components/style/Font.js:8]:
```suggestion
export default function Font() {
```

File: components/style/Reset.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { COLORS } from '../../lib/constants'
 
-export default () => (
+const Reset = () => (
```
Comment (mfix22) [inline components/style/Reset.js:4]:
```suggestion
export default function Reset() {
```

File: components/style/Typography.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 
-export default () => (
+const Typography = () => (
```
Comment (mfix22) [inline components/style/Typography.js:3]:
```suggestion
export default function Typography() {
```

File: components/svg/Language.js [hunk 5]
```diff
@@ -1,10 +1,11 @@
 import React from 'react'
 
-export default () => (
+const Language = () => (
```
Comment (mfix22) [inline components/svg/Language.js:3]:
```suggestion
export default function Language() {
```

File: components/svg/Logo.js [hunk 6]
```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 
-export default () => (
+const Logo = () => (
```
Comment (mfix22) [inline components/svg/Logo.js:3]:
```suggestion
export default function Logo() {
```

File: components/svg/Settings.js [hunk 7]
```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 
-export default () => (
+const Settings = () => (
```
Comment (mfix22) [inline components/svg/Settings.js:3]:
```suggestion
export default function Settings() {
```

File: components/svg/Theme.js [hunk 8]
```diff
@@ -1,10 +1,11 @@
 import React from 'react'
 
-export default () => (
+const Theme = () => (
```
Comment (mfix22) [inline components/svg/Theme.js:3]:
```suggestion
export default function Theme() {
```

File: pages/about.js [hunk 9]
```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import Page from '../components/Page'
 
-export default () => (
+const About = () => (
```
Comment (mfix22) [inline pages/about.js:4]:
```suggestion
export default function About() {
```

File: pages/snippets.js [hunk 10]
```diff
@@ -237,9 +242,10 @@ function SnippetsPage() {
   )
 }
 
-export default () => (
+const Snippets = () => (
```
Comment (mfix22) [inline pages/snippets.js:245]:
```suggestion
export default function Snippets() {
```

File: pages/snippets.js [hunk 11]
```diff
@@ -25,7 +25,12 @@ function correctTimestamp(n) {
 }
 
 function Snippet(props) {
-  const config = { ...DEFAULT_SETTINGS, ...props, fontSize: '2px', windowControls: false }
+  const config = {
```
Comment (mfix22) [inline pages/snippets.js:28]: Please keep this as is, unless prettier adjusted this for you.

File: components/Billing.js [hunk 12]
```diff
@@ -144,125 +144,127 @@ function Billing(props) {
...
-          color: ${COLORS.RED};
-        }
-
-        .success {
-          font-size: 16px;
-          line-height: 1.5;
-          margin: 0 0 2rem;
-        }
-      `}</style>
+      <style jsx>
```
Comment (mfix22) [inline components/Billing.js:147]: Please keep this as is, unless Prettier adjusted this for you

**Review by mfix22 — CHANGES_REQUESTED** [review]
@Aslemammad Thanks so much! Just have some style adjustments I'd like to make, but besides that this is great

**Comment by Aslemammad** [comment]
@mfix22 I made the new changes. Thanks.

File: components/Billing.js [hunk 13]
```diff
@@ -144,125 +144,127 @@ function Billing(props) {
...
+        `}
+      </style>
     </div>
   )
 }
 
 const BillingWithStripe = injectStripe(Billing)
 
-export default () => {
+export default function ExportedBilling() {
```
Comment (mfix22) [inline components/Billing.js:267]:
```suggestion
export default function BillingPage() {
```

File: pages/snippets.js (hunk 11 above)
Comment (mfix22) [reply pages/snippets.js:28]: Is this from Prettier? Or can you keep this as is?

**Review by mfix22 — CHANGES_REQUESTED** [review]
@Aslemammad a couple more changes! 

Other open issues: 
* https://github.com/carbon-app/carbon/issues?q=is%3Aissue+is%3Aopen+label%3AHacktoberfest
* https://github.com/carbon-app/carbon/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22+

**Comment by Aslemammad** [comment]
No, it's from prettier config.

**Comment by Aslemammad** [comment]
Thanks for merge.

**Pull request merged**